src/PlatformClient.php

```diff
@@ -333,4 +338,14 @@ public function getRegions()
     {
         return Region::getCollection($this->accountsEndpoint . 'regions', 0, [], $this->getConnector()->getClient());
     }
+    
+    /**
+     * Get the project options catalog.
+     *
+     * @return Catalog[]
+     */
+    public function getCatalog()
+    {
+        return Catalog::create([], $this->accountsEndpoint . 'setup/catalog', $this->getConnector()->getClient());
```

Comment:
Pretty much the same objections as #39 - the catalog isn't a thing that lists items on GET /catalog, lets me GET a consitutuent item in GET/catalog/:id, etc, DELETE/PUT/PATCH to modify, POST to add to the list, etc, it's not a RESTful resource, so reusing the Resource model here is at least a bit awkward.